src/privatebin.rs

```diff
@@ -30,11 +33,12 @@ pub enum PasteFormat {
     Markdown,
 }
 
-#[derive(Deserialize, Debug, Serialize)]
+#[skip_serializing_none]
+#[derive(Default, Deserialize, Debug, Serialize)]
 pub struct Paste {
-    pub status: i32,
+    pub status: Option<i32>,
     pub id: String,
-    pub url: String,
+    pub url: Option<String>,
```

Comment:
Would changing this to an URL make sense or does that interfere with uniffi?

Reply:
There would be no issue with uniffi. 
But the "url" returned by privatebin post response can be weird:

```json
{"status":0,"id":"ba7bf946a542eee1","url":"\/?ba7bf946a542eee1","deletetoken":"9914b0e934253e2e327f9a6477b99f6fff5d1e0e10d3d3bd08d4d9b6a20ab2ea"}
```

So just need to test if it parses correctly.  
Same with above.

Reply:
Have opened an issue upstream regarding this https://github.com/PrivateBin/PrivateBin/issues/1294

Reply:
Hey, 

in that case it appears there is nothing that can be done from our side, so I will proceed and merge this. Thank you for the contribution!

Best Regards
Mydayyy